brainscore/benchmarks/hebart2023.py

```diff
@@ -0,0 +1,39 @@
+from brainscore.benchmarks import BenchmarkBase
+from brainscore.model_interface import BrainModel
+
+
+BIBTEX = """@article{10.7554/eLife.82580,
+          author = {Hebart, Martin N and Contier, Oliver and Teichmann, Lina and Rockter, Adam H and Zheng, Charles Y and Kidder, Alexis and Corriveau, Anna and Vaziri-Pashkam, Maryam and Baker, Chris I},
+          journal = {eLife},
+          month = {feb},
+          pages = {e82580},
+          title = {THINGS-data, a multimodal collection of large-scale datasets for investigating object representations in human brain and behavior},
+          volume = 12,
+          year = 2023
+          }"""
+
+class _Hebart2023Accuracy(BenchmarkBase):
+    # behavioral benchmark
+    def __init__(self, dataset):
```

Comment:
```suggestion
class Hebart2023Accuracy(BenchmarkBase):
    def __init__(self):
```
You can simplify this unless you are planning different versions of the same benchmark type.